app/services/github_repository_client.rb

```diff
@@ -18,4 +16,18 @@ def repository_views
     response = connection.get { |request| request.params['per'] = 'week' }
     JSON.parse(response.body).with_indifferent_access
   end
+
+  private
+
+  def find_in_locations
+    content = LOCATIONS.each do |location|
+      connection = Faraday.new(url: "#{URL}/#{@project_name}/contents/#{location}") do |conn|
+        conn.basic_auth(ENV['GITHUB_ADMIN_USER'], ENV['GITHUB_ADMIN_TOKEN'])
+      end
+
+      response = connection.get { |req| req.headers['Accept'] = 'application/vnd.github.v3.raw' }
+      return response.body if response.success?
+    end
+    content.is_a?(Array) ? '' : content
```

Comment:
What is this line supposed to do?

Reply:
if the array doesn't return anything, stores the initial array of locations in the variable so when that happens we want to return an empty string instead of the array of locations

Reply:
In that case, why are we checking what's in `content`? If we made it to this line, then the file couldn't be found in any location. In which case we should just return `''`
Does it make sense?